.github/workflows/prod-deploy-api.yml

```diff
@@ -63,23 +73,25 @@ jobs:
           REGISTRY_OWNER: novuhq
           DOCKER_NAME: ${{matrix.name}}
           IMAGE_TAG: ${{ github.sha }}
+          SEMVER_IMAGE_TAG: ${{ needs.get_version.outputs.current_tag }}-prod
```

Comment:
Why is this ```-prod```

Reply:
If there is an infra change we need to make to fully support semver let me know.

Reply:
It's a temporary solution until we can exclude a docker build for a production environment. The idea is to build the image in the beginning and after use the already built and tested image to avoid possible issues. At the current moment, we have no chance to use equal tags for both environments because the docker build step will replace the image that was created earlier.